README.md

```diff
@@ -95,8 +98,27 @@ kind get kubeconfig > ./kube.config
 
 ### How does it work
 
-To better depict what `ats` does and how it executes a test scenario, let's have a look at what happens
-when you execute:
+Each run consist of two stages: bootstrapping and test execution.
+
+#### Bootstrapping
+
+`app-test-suite` automates preparation of a cluster used for testing in the following way:
+
+- if you configured your run with `*-tests-cluster-type kind`, a cluster is created with `kind` tool
+- `ats` connects to the target test cluster and runs `apptestctl` - additional tool that deploys components of App Platform
+  (`app-operator`, `chart-operator` and needed CRDs)
```

Comment:
Could "needed CRDs" be made more specific?

In the case I'm struggling with currently, my Kyverno policies repo needs Kyverno and a Kyverno CRD. So I'm wondering "how does apptestctl know that the Kyverno CRDs are required?". I'm guessing that here, "needed CRDs" is simply referring to the app platform standard, right?

Reply:
Yes, it's just more details about the App Platform part